### PR TITLE
add illustration server

### PR DESCRIPTION
**Pull request by rockitbaby**
The illustrator server renders the product field
as svg image. I will be used to create explaining
images for the product field (e-)book.

references:
- https://github.com/webpack/react-webpack-server-side-example
- https://facebook.github.io/react/docs/top-level-api.html#reactdomserver.rendertostaticmarkup
- https://webpack.github.io/docs/multiple-entry-points.html
- http://jxnblk.com/react-icons/

---

given this new need (to render static product field images), it makes sense to restructure the code-base as suggested.
- App
  - Editor
  - ForceField
    - Stage
    - Forces
    - (../../Shared/Renderer)
- Shared
  - Renderer
    - Grid
    - Marker
- Illustrator
  - Server
  - (../Shared/Renderer)

### CODE REVIEW

**Comment by rockitbaby** [comment]
hey @mortik @sebashwa!
i'll add / refactor some drawing code today to match the illustration needs. let's decide together, when to refactor the overall structure, merge this pull request.

**Comment by rockitbaby** [comment]
example svg: https://productfield.herokuapp.com/?width=1280&height=600

**Comment by mitch000001** [comment]
@rockitbaby @mortik and myself have rebased the branch so you can check it out and see if everything is still working fine. Afterwards it's all green to merge.

**Comment by rockitbaby** [comment]
awesome! thank you
Am 05.02.2016 8:47 nachm. schrieb "Michael Wagner" <notifications@github.com

File: README.md [hunk 1]
```diff
@@ -15,7 +15,13 @@ A Force Field Analysis Tools for the Product Field Canvas.
 - `npm run devserver`
 - visit `http://localhost:8080/`
 
-#### ComponentLibrary
+### Run Illustrator Server
+
+The illustrator server renders the product field as svg image. I will be used to create explaing images fpr the product field (e-)book.
```
Comment (JoschkaSchulz) [inline README.md:20]: typo "[...] explaing images fpr [...]" should be "[...] explaining images for [...]"

**Pull request merged**